core/storage/src/trie/mod.rs

```diff
@@ -854,30 +841,28 @@ mod tests {
         assert_eq!(node, new_node);
     }
 
-    #[test]
-    fn test_basic_trie() {
-        let storage: Arc<KeyValueDB> = Arc::new(create_memory_db());
-        let trie = Trie::new(storage.clone());
-        let empty_root = Trie::empty_root();
-        // assert_eq!(trie.get(&empty_root, &[122]), None);
-        let changes = vec![
-            (b"doge".to_vec(), Some(b"coin".to_vec())),
-            (b"docu".to_vec(), Some(b"value".to_vec())),
-            (b"do".to_vec(), Some(b"verb".to_vec())),
-            (b"horse".to_vec(), Some(b"stallion".to_vec())),
-            (b"dog".to_vec(), Some(b"puppy".to_vec())),
-            (b"h".to_vec(), Some(b"value".to_vec())),
-        ];
-        let root = test_populate_trie(&trie, &empty_root, changes.clone());
-        let new_root = test_clear_trie(&trie, &root, changes);
-        assert_eq!(new_root, empty_root);
-        assert_eq!(storage.iter(Some(0)).fold(0, |acc, _| acc + 1), 0);
-    }
+//    #[test]
```

Comment:
bring back?

Reply:
Now, I remember why. This unittests verifies some trie stuff by accessing the storage column directly. I have commented out that specific assertion.